README.md

```diff
@@ -32,8 +30,7 @@ of differential equations,
 Together, these enable flexible model definitions and fast battery simulations, allowing users to
 explore the effect of different battery designs and modeling assumptions under a variety of operating scenarios.
 
-[//]: # "numfocus-fiscal-sponsor-attribution"
-
+[//]: #
```

Comment:
This should be reverted. IIRC NumFOCUS asked us to add this because their automated system scans look for this line in every sponsored project's README.

Reply:
Noted, will do.
Again might e done while merging